src/dmd/dsymbolsem.d

```diff
@@ -1978,15 +1978,20 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             }
         }
 
-        ed.semanticRun = PASS.semanticdone;
+        if (!(sc.flags & SCOPE.Cfile))  // C enum remains incomplete until members are done
+            ed.semanticRun = PASS.semanticdone;
```

Comment:
Would make more sense to move this to line 1997.

Reply:
ok